### DISCUSSION TITLE
Info about Vaultwarden security

### DISCUSSION THREAD

**Discussion by lukethehawk** [dani-garcia/vaultwarden#3622] [General]
Hello!
Considering that Vaultwarden is several versions behind Bitwarden (web vault), I was wondering if this could potentially pose greater cybersecurity risks. We are currently using version 2023.3.0 of Vaultwarden and Bitwarden is 2023.5.0. So my question is, could there be a risk of security vulnerabilities?
Thank you

**Comment by BlackDex** [comment]
For that i suggest to read the changelogs of the web-vault client.

If you want to use the latest web-vault version i suggest to use the `:testing` tagged docker images.

**Comment by vermium** [comment]
I'm interested on this topic too
